src/phoebus_driver.cpp

```diff
@@ -698,11 +698,13 @@ TaskListStatus PhoebusDriver::RadiationPostStep() {
     if (do_lightbulb) {
       pdo_gain_reducer = rad->MutableParam<parthenon::AllReduce<bool>>("do_gain_reducer");
     }
-    TaskRegion &async_region = tc.AddRegion(num_independent_task_lists);
-    for (int ib = 0; ib < num_independent_task_lists; ib++) {
-      auto pmb = blocks[ib].get();
-      auto &tl = async_region[ib];
-      auto &sc0 = pmb->meshblock_data.Get(stage_name[integrator->nstages]);
+    // creating a new sync region for light bulb functions
+    const int num_partitions = pmesh->DefaultNumPartitions();
+    TaskRegion &sync_region_lb = tc.AddRegion(num_partitions);
```

Comment:
Change "_lb" to "_rad"

Reply:
Nope

Reply:
In case, you've stumbled upon this, we're joking.

Reply:
I figured